.github/workflows/workspace-release-e2e-workflow.yml

```diff
@@ -0,0 +1,34 @@
+name: Workspace Release tests workflow in Bundled OpenSearch Dashboards
+on:
+  pull_request:
+    branches: ['**']
+
+jobs:
+  changes:
+    runs-on: ubuntu-latest
+    outputs:
+      tests: ${{ steps.filter.outputs.tests }}
+    steps:
+      - uses: dorny/paths-filter@v2
+        id: filter
+        with:
+          filters: |
+            tests:
+              - 'cypress/**/dashboards-workspace/**'
```

Comment:
The test pattern should match `cypress/integration/core-opensearch-dashboards/opensearch-dashboards/workspace-plugin` I think?